src/plugins/condition/components/inspector/StyleEditor.vue

```diff
@@ -216,6 +234,13 @@ export default {
             }
 
             this.$emit('persist', this.styleItem, item.property);
+        },
+        canSaveStyle() {
+            return this.isEditing && !this.mixedStyles.length && !this.nonSpecificFontProperties.length;
+        },
+        saveItemStyle() {
+            // this.stylesManager.save(this.itemStyle);
```

Comment:
remove commented code

Reply:
yup. removing this and injection of stylesManager as that work needs to be pushed to StylesView